src/cmd/initialize.go

```diff
@@ -156,24 +171,24 @@ func init() {
 
 	// Continue to require --confirm flag for init command to avoid accidental deployments
 	initCmd.Flags().BoolVar(&config.CommonOptions.Confirm, "confirm", false, "Confirm the install without prompting")
-	initCmd.Flags().StringVar(&config.InitOptions.Components, "components", v.GetString(V_INIT_COMPONENTS), "Comma-separated list of components to install.")
-	initCmd.Flags().StringVar(&config.InitOptions.StorageClass, "storage-class", v.GetString(V_INIT_STORAGE_CLASS), "Describe the StorageClass to be used")
+	initCmd.Flags().StringVar(&pkgConfig.InitOpts.Components, "components", v.GetString(V_INIT_COMPONENTS), "Comma-separated list of components to install.")
```

Comment:
As discussed (in person-ish), this PR will set the baseline for lang file capability and we'll slowly go about utilizing the lang file as we complete future work instead of also trying to cram that into this PR.